app/assets/javascripts/app/app.js

```diff
@@ -148,6 +149,17 @@ var app = {
       $(this).clearForm();
       $(this).focusout();
     });
+  },
+
+  setupAjaxErrorRedirect: function() {
+    var self = this;
+    $(document).ajaxError(function(evt, jqxhr) {
+      if(jqxhr.status === 401) { self._changeLocation(Routes.newUserSession()); }
+    });
+  },
+
+  _changeLocation: function(href) {
+    window.location.assign(href);
   }
```

Comment:
This is the only way I found which allows us to test the behavior with jasmine. The tests would fail as soon as we really assign a new location.